pkg/ui/src/util/events.ts

```diff
@@ -48,6 +49,12 @@ export function getEventDescription(e: Event$Properties): string {
       return `Schema Change: User ${info.User} began a schema change to alter index ${info.IndexName} on table ${info.TableName} with ID ${info.MutationID}`;
     case eventTypes.CREATE_VIEW:
       return `View Created: User ${info.User} created view ${info.ViewName}`;
+    case eventTypes.CREATE_SEQUENCE:
+      return `Sequence Created: User ${info.User} created sequence ${info.SequenceName}`;
+    case eventTypes.ALTER_SEQUENCE:
+      return `Sequence Altered: User ${info.User} altered sequence ${info.SequenceName}`;
+    case eventTypes.DROP_SEQUENCE:
+      return `Sequence Dropped: User ${info.User} dropped sequence ${info.SequenceName}`;
```

Comment:
looks like the new events are between create view and drop view

Reply:
D'oh. Fixed.